libs/model/src/feed/GetUserActivity.query.ts

```diff
@@ -1,63 +1,84 @@
 import { Query } from '@hicommonwealth/core';
 import * as schemas from '@hicommonwealth/schemas';
 import { QueryTypes } from 'sequelize';
+import { z } from 'zod';
 import { models } from '../database';
 
+/**
+ * Last 50 updated threads
+ */
 export function GetUserActivity(): Query<typeof schemas.ThreadFeed> {
   return {
     ...schemas.ThreadFeed,
     auth: [],
     secure: true,
     body: async ({ actor }) => {
-      /**
-       * Last 50 updated threads
-       */
-
-      const filterByCommunityForUsers = actor.user.id
-        ? 'JOIN "Addresses" a on a.community_id=t.community_id and a.user_id = ?'
-        : '';
-
       const query = `
-    WITH ranked_thread_notifs AS (
-        SELECT t.id AS thread_id, t.max_notif_id
-        FROM "Threads" t
-        ${filterByCommunityForUsers}
-        WHERE deleted_at IS NULL
-        ORDER BY t.max_notif_id DESC
-        LIMIT 50
-    ) SELECT
-        nts.thread_id,
-        nts.created_at AS last_activity,
-        nts.notification_data,
-        nts.category_id,
-        thr.comment_count,
-                COALESCE(
-                    json_agg(
-                        json_build_object('Addresses', json_build_array(row_to_json(A)))
-                    ) FILTER (WHERE A.id IS NOT NULL),
-                    json_build_array()
-                ) as commenters
-    FROM ranked_thread_notifs rtn
-    INNER JOIN "Notifications" nts ON rtn.max_notif_id = nts.id
-    JOIN "Threads" thr ON thr.id = rtn.thread_id
-    LEFT JOIN "Comments" C ON C.id = (nts.notification_data::JSONB ->> 'comment_id')::INTEGER
-    LEFT JOIN LATERAL (
-        SELECT A.id, A.address, A.community_id, A.profile_id
```

Comment:
I guess this is fine since we are not using this query on the front end yet, but basically the in the REST query that we do use (GET /viewUserActivity), we use the profile_id to link the user's profile. 

Was there some solution we came up with in your discussion with @masvelio on the front end?

Reply:
Also, why are we modifying this query and not the query that is currently used for the feeds? There is no point in consistently maintaining this route in parallel to the in-use activity routes if we don't switch to tRPC on the client. #8448 technically should have modified this query (but didn't) so the modification in this PR won't be valid since this query is already out of date.

Reply:
I'm just refactoring everything that references Profiles... we can remove deprecated routes/queries later or if we are all 100% sure it can be removed... to me everything that. ends with .query is new work.